utils/driverless.1

```diff
@@ -48,10 +48,18 @@ Debug mode, verbose logging to stderr.
 .TP
 .B
 \fBlist\fP
-List the driver URIs and metadata for all available IPP printers suitable
+List the driver URIs and metadata for all available IPP/IPPS printers suitable
 for driverless PPD file generation (to be used by CUPS).
 .TP
 .B
+\fB_ipps._tcp\fP
+Check for only IPPS printers supporting driverless printing (to be  used by CUPS).
+.TP
+.B
+\fB_ipp._tcp\fP
+Check for only IPP printers supporting driverless printing (to be  used by CUPS).
```

Comment:
Note that these command line arguments are only used for the manual call of `driverless` from a terminal windows. The remark "(to be used by CUPS)" in the description of the argument means that the argument is used by CUPS, like the `list` and `cat` arguments. So could you please remove the "(to be used by CUPS)" for these two arguments.

Reply:
Sure, I would made the respective changes in driverless.c file also ,to use these command  line arguments  only for the manual call of `driverless ` from a terminal windows.

Reply:
You mean in the help output? Yes, please remove it there, too.